neo/rawio/plexon2rawio/plexon2rawio.py

```diff
@@ -74,14 +76,24 @@ def __init__(self, filename, pl2_dll_file_path=None):
 
         # download default PL2 dll once if not yet available
         if pl2_dll_file_path is None:
-            pl2_dll_folder = pathlib.Path .home() / '.plexon_dlls_for_neo'
+            architecture = platform.architecture()[0]
+            if architecture == '64bit':
+                file_name = "PL2FileReader64.dll"
+            else:
+                file_name = "PL2FileReader.dll"
+            
+            pl2_dll_folder = pathlib.Path.home() / '.plexon_dlls_for_neo'
             pl2_dll_folder.mkdir(exist_ok=True)
-            pl2_dll_file_path = pl2_dll_folder / 'PL2FileReader.dll'
+            pl2_dll_file_path = pl2_dll_folder / file_name
 
-            if pl2_dll_file_path.exists():
-                warnings.warn(f'Using cached plexon dll at {pl2_dll_file_path}')
+            if not pl2_dll_file_path.exists():
```

Comment:
I'm confused why did you switch to the opposite logic here?

Reply:
Or did you mean to download with urlopen if it doesn't exist and then just delete the else since the warning is likely not needed.

Or maybe I'm just misunderstanding the flow?

Reply:
No, that's a mistake. Thanks.

I removed the warning beforehand and then re-verted without re-re-verting the logic.

Reply:
Ah cool cool. That makes sense.